net.sonobus.SonoBus.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: net.sonobus.SonoBus
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '20.08'
+command: sonobus
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home
+# Jack
+  - --filesystem=xdg-run/pipewire-0
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+
+modules:
+  - name: jack2
+    buildsystem: simple
+    build-commands:
+      - ./waf configure --prefix=/app --htmldir=/app/share/doc/jack/ --classic
+      - ./waf build -j $FLATPAK_BUILDER_N_JOBS
+      - ./waf install
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://github.com/jackaudio/jack2/archive/refs/tags/v1.9.17.tar.gz
+        sha256: 38f674bbc57852a8eb3d9faa1f96a0912d26f7d5df14c11005ad499c8ae352f2
```

Comment:
You can include [JACK from shared-modules](https://github.com/flathub/shared-modules/blob/b0d4c06751d0451c0a6cd9672b3132d6152f332c/linux-audio/jack2.json#L1-L21) if you want to reduce some code here.

Instructions: https://github.com/flathub/flathub/wiki/App-Requirements#shared-modules

Reply:
Thanks, will do that.